### DISCUSSION TITLE
There is a major outage with GitHub Actions since at least 2 hours

### DISCUSSION THREAD

**Discussion by SekouDiaoNlp** [juancarlospaco/faster-than-csv#25] [General]
See [Github Actions Status](https://www.githubstatus.com/)

![githubstatus](https://user-images.githubusercontent.com/52342703/118391985-62aea000-b637-11eb-8086-530c417010d7.png)

**Comment by SekouDiaoNlp** [comment]
Just when I was modifying the workflows to run with nim 1.2.12 on MacOs for now as per https://github.com/Pebaz/nimporter/pull/48

😞 😭

**Comment by SekouDiaoNlp** [comment]
It is getting even worse 👎🏿 

![githubstatus2](https://user-images.githubusercontent.com/52342703/118395432-baa2d200-b64a-11eb-8d6f-a1895eee9a9e.png)

**Comment by juancarlospaco** [comment]
https://therecord.media/github-investigating-crypto-mining-campaign-abusing-its-server-infrastructure

**Reply by SekouDiaoNlp** [reply]
WTH!
